inappmessaging/src/main/java/com/rakuten/tech/mobile/inappmessaging/runtime/manager/DisplayManager.kt

```diff
@@ -11,6 +11,9 @@ import timber.log.Timber
  * Display manager, which controls displaying message, or removing message from the screen.
  */
 internal interface DisplayManager {
+
+    var verifyCampaignContextsCallback: (contexts: List<String>, campaignTitle: String) -> Boolean
```

Comment:
We can just access the `onVerifyContext` from `InAppMessaging.instace()` from `DisplayMessageJobIntentService` instead of passing the callback to `DisplayManager`

Reply:
Is that ok? To just arbitrarily call some method from singleton. It sounds like a violation of clean architecture rules but maybe it's ok here. In that case I don't want to comlpicate the code

Reply:
👍